packages/vanilla/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    "es2015",
+    "stage-2"
+  ],
+  "plugins": []
```

Comment:
Note that I'm leaving this to the default option - `"compact": "auto"`, which seems like what we want?

Reference: https://babeljs.io/docs/usage/api/#options

Reply:
that sound good.